sidebars.js

```diff
@@ -843,13 +843,46 @@ const sidebars = {
           type: "category",
           label: "Tokens",
           items: [
-            "developer-docs/defi/tokens/token-standards",
+	          "developer-docs/defi/tokens/token-standards",
+	          "developer-docs/defi/tokens/asset_flow/index",
+            {
+              type: "category",
+              label: "Ledgers",
+              link: {
+                type: "doc",
+                id: "developer-docs/defi/tokens/ledger/overview",
+              },
+              items: [
+                {
+                  type: "category",
+                  label: "Setup",
+                  link: {
+                    type: "doc",
+                    id: "developer-docs/defi/tokens/ledger/setup/overview",
+                  },
+                  items: [
+                    "developer-docs/defi/tokens/ledger/setup/icp_ledger_setup",
+                    "developer-docs/defi/tokens/ledger/setup/icrc1_ledger_setup"
+                  ],
+                },
+                {
+                  type: "category",
+                  label: "Usage",
+                  link: {
+                    type: "doc",
+                    id: "developer-docs/defi/tokens/ledger/usage/overview",
+                  },
+                  items: [
+                    "developer-docs/defi/tokens/ledger/usage/icp_ledger_usage",
+                    "developer-docs/defi/tokens/ledger/usage/icrc1_ledger_usage"
+                  ],
+                },
+                "developer-docs/defi/tokens/advanced/direct_integration",  
+              ],
+            },
+            
             "developer-docs/defi/token_integrations/index",
```

Comment:
```suggestion

```

Reply:
This is a duplicate of `developer-docs/defi/tokens/advanced/direct_integration` - it contains the same content. It should be removed from this sidebars.js file and the `developer-docs/defi/token_integrations/index` file itself should be removed as well.